cmd/gcp-controller-manager/app/node_annotater.go

```diff
@@ -105,11 +105,14 @@ func newNodeAnnotator(client clientset.Interface, nodeInformer coreinformers.Nod
 			{
 				name: "labels-reconciler",
 				annotate: func(node *core.Node, instance *compute.Instance) bool {
-					labels, err := extractKubeLabels(instance)
+					labels, ok, err := extractKubeLabels(instance)
 					if err != nil {
 						glog.Errorf("Error reconciling labels: %v", err)
 						return false
 					}
+					if !ok {
+						return false
```

Comment:
true?

Reply:
If we don't find metadata, we don't need to update labels and we also don't need to log an error.